### ISSUE TITLE
Webcam stream enabled by default

### ISSUE THREAD

**Issue by cyb0man** [OctoPrint/OctoPrint#3645]
#### What were you doing?
I have noticed that Octoprint by default will eat a lot of bandwidth, 500-700 kbps.
This was a normal install on a RPi.
No camera connected.

#### What did you expect to happen?
I would suggest that camera stream is disabled by default, especially stupid if there is no camera connected.

#### What happened instead?
I disabled the feature and made reboot.
(Restart did not help!)

#### Version of OctoPrint
1.4.0

#### Operating System running OctoPrint
Linux OctoPi 4.19.118+ 

#### Screenshot(s)/video(s) showing the problem:

I have read the FAQ.

**Comment by gdachs** [comment]
OctoPrint is not involved in the webcam streaming. That is done by mjpg-streamer that is not part of OctoPrint.

**Comment by foosel** [comment]
Please fill out the *complete* ticket template and explain your issue a bit more. There's nothing to stream if there's no webcam connected, so either your bandwidth comes from something else (third party plugin? **Test in safe mode**) or there's a webcam connected after all - which btw only gets loaded if the control tab is active, unless you run some wonky browser that doesn't support that behaviour.

In short: It's completely unclear what your problem actually is and you didn't provide enough information to determine that.

**Comment by foosel** [comment]
Closing due to lack of response.
